### PR TITLE
Fixed dependence between Load Balancer and ECS

### PR DESCRIPTION
**Pull request by luizfdsilva**

### CODE REVIEW

File: terraform/modules/aws-ecs-task-def/main.tf [hunk 1]
```diff
@@ -51,17 +51,22 @@ resource "aws_ecs_task_definition" "project_1" {
...
+    efs_volume_configuration {
+      file_system_id = var.file_system_id
+      root_directory = "/opt/data"
     }
+
+    # docker_volume_configuration {
+    #   scope         = var.volume_scope
+    #   autoprovision = var.volume_autoprovision
+    #   driver        = var.volume_driver
+
```
Comment (christina-roperto) [inline terraform/modules/aws-ecs-task-def/main.tf]: I think we should avoid putting commented out codes for master

File: terraform/modules/aws-ecs-service/main.tf [hunk 2]
```diff
@@ -16,4 +17,6 @@ resource "aws_ecs_service" "ecs" {
     container_name   = var.container_name
     container_port   = var.container_port
   }
+
+  depends_on = [var.ecs_service_depends_on]
```
Comment (christina-roperto) [inline terraform/modules/aws-ecs-service/main.tf:21]: good stuff Luiz

**Review by christina-roperto — DISMISSED** [review]
Good stuff Luiz, if you can remove the comments just like what Denis mentioned on the meeting, it would be great!

**Comment by luizfdsilva** [comment]
> Good stuff Luiz, if you can remove the comments just like what Denis mentioned on the meeting, it would be great!

Thanks! I removed the comments now.

**Review by christina-roperto — APPROVED** [review]

**Pull request merged**